AST WhileStmt has 2 children in Clang 9.0 (instead of 3 children)

The `WhileStmt` node of Clang 9.0 has 2 children nodes, rather than 3 as in prior versions of Clang.

This results in a panic `transpileWhileStmt`, as presented below.

## Example source file

```c
int f(int x) {
	int sum = 0;
	int i = 0;
	while (i < x) {
		sum += i;
		i++;
	}
	return sum;
}
```

## Panic

```
$ c2go transpile i.c
panic: runtime error: index out of range [2] with length 2

goroutine 1 [running]:
github.com/elliotchance/c2go/transpiler.transpileWhileStmt(0xc0000c08a0, 0xc0000e4000, 0x8, 0xc0003209b0, 0x10, 0x10, 0xc0000a7140, 0x0, 0xc0000a7130, 0x8b37a0, ...)
```

--- a/transpiler/branch.go
+++ b/transpiler/branch.go
@@ -439,17 +439,22 @@ func transpileForStmt(n *ast.ForStmt, p *program.Program) (
 //    |     `-DeclRefExpr 0x2530c78 <col:3> 'int' lvalue Var 0x25306f8 'i' 'int'
 func transpileWhileStmt(n *ast.WhileStmt, p *program.Program) (
 	goast.Stmt, []goast.Stmt, []goast.Stmt, error) {
+	// Skip first nil-child if present.
+	children := n.Children()
+	if len(children) == 3 && children[0] == nil {
+		children = children[1:]
+	}
 	var forOperator ast.ForStmt
 	forOperator.AddChild(nil)
 	forOperator.AddChild(nil)
-	forOperator.AddChild(n.Children()[1])
+	forOperator.AddChild(children[0])
 	forOperator.AddChild(nil)
-	if n.Children()[2] == nil {
+	if children[1] == nil {
 		// added for case if WHILE haven't body, for example:
 		// while(0);
-		n.Children()[2] = &ast.CompoundStmt{}
+		children[1] = &ast.CompoundStmt{}
 	}
-	forOperator.AddChild(n.Children()[2])
+	forOperator.AddChild(children[1])
 
 	return transpileForStmt(&forOperator, p)
 }

